content/docs/user-guide/gems/reference/multiplayer/multiplayer/tutorial_multiplayer.md

```diff
@@ -0,0 +1,887 @@
+---
+title: Tutorial - Your First Network Component
+linktitle: Tutorial Multiplayer
+description: Use Multiplayer Gem to create multiplayer components.
+toc: true
+---
+
+
+
+## Introduction
+
+
+This tutorial will link all the relevant documentation to provide you with a detailed guide on getting started with creating multiplayer components.
+
+Our starting base will be O3DE MultiplayerSample project. You can find the instructions on how to build and run it here: https://github.com/o3de/o3de-multiplayersample/blob/development/README.md
+
+
+
+
+
+## Network Components
+
+### Add a New Component
+
+We will start by creating a network component - MyFirstNetworkComponent. It will only do one thing, which is to replicate a monotonic counter.
+
+Since we already have MultiplayerSample setup for codegen, we can start by duplicating one of the existing codegen xml files.
+
+`D:\git\o3de-multiplayersample\Gem\Code\Source\AutoGen\SimplePlayerCameraComponent.AutoComponent.xml`
```

Comment:
Unsure if there is a standard for this but do we want the full path as above or a relative one that starts with o3de-multiplayersample?

Reply:
That is a good question. My view is that the path doesn't show any personal details from my computer and could apply to everyone, and I don't have to deal with the question: is the path inside the project or inside the engine. But maybe \path\to\o3de-multiplayersample is better than D:\git\o3de-multiplayersample ?

Reply:
I'll go with \path\to\o3de-multiplayersample for now.

Reply:
I like that a lot actually